### PR TITLE
Fix typo and crontab issue on wcoss dell in workflow python scripts

### PR DESCRIPTION
**Pull request by chan-hoo**
## DESCRIPTION OF CHANGES: 
- Fix typo in 'setup.py'.
- Fix crontab issue on wcoss_dell_p3 in the workflow python scripts.
- Remove capture_output as PR #744.

## TESTS CONDUCTED: 
- 6h GST tests with FV3GFS_v15p2 and FV3GFS_v16 on wcoss_dell_p3

### CODE REVIEW

File: ush/get_crontab_contents.py [hunk 1]
```diff
@@ -40,7 +40,10 @@ def get_crontab_contents(called_from_cron):
     #import all env vars
     IMPORTS = ["MACHINE"]
     import_vars(env_vars=IMPORTS)
-  
+
+    # Get username on current machine
+    global USER
```
Comment (danielabdi-noaa) [inline ush/get_crontab_contents.py]: A minor comment: `USER` doesn't have to be global unless we want to export its value here and use it in other files as well. Maybe it is a good idea to define env variables that come pre-defined with shell, such as `USER`, `PATH` etc in one central place, similar to the way `SED` is defined in `define_macos_utilities.py`

**Review by danielabdi-noaa — APPROVED** [review]
@chan-hoo Thanks for fixes. It looks good to me so I will approve!
I suppose you were able to run some test cases after these fixes, right? WCOSS has not been tested before with the python workflow generator.

File: ush/get_crontab_contents.py (hunk 1 above)
Comment (chan-hoo) [reply ush/get_crontab_contents.py]: @danielabdi-noaa, I'll try it and let you know if I have any problems.

Comment (chan-hoo) [reply ush/get_crontab_contents.py]: @danielabdi-noaa, I put "set_env_var('USER','USER')" in `define_macos_utilities.py`, but it didn't work, I got  "   (_,__crontab_contents__,_)=run_command(f'''cat "/u/{USER}/cron/mycrontab"''')
NameError: name 'USER' is not defined". I don't think I understood your comment correctly. Can you make this change on your end? I've sent a github collaborator invite to you.

Comment (danielabdi-noaa) [reply ush/get_crontab_contents.py]: I think `USER`, `HOME` etc were already in the environment so I just added it to the IMPORTS list at the beginning of `get_crontab_contents.py`. Let me know if that works now. Thanks!

Comment (chan-hoo) [reply ush/get_crontab_contents.py]: Thank you! I'll test it and let you know.

Comment (chan-hoo) [reply ush/get_crontab_contents.py]: @danielabdi-noaa, it worked! Thank you for your help!

**Review by gsketefian — APPROVED** [review]

**Pull request merged**